manifests/init.pp

```diff
@@ -35,12 +35,17 @@
 #
 # [*purge_config_dir*]
 #   Purge config files no longer generated by Puppet
+#
+# [*pretty_print_json*]
+#   Add whitespace to config to make it more readable. Defaults to `false`.
+#
 class consul (
   $manage_user           = true,
   $user                  = 'consul',
   $manage_group          = true,
   $extra_groups          = [],
   $purge_config_dir      = true,
+  $print_pretty_json     = false,
```

Comment:
For consistency sake, can we name all these vars `pretty_config` (in `config.pp` as well)? Currently I see `pretty_print_json`, `print_pretty_json` and `pretty` lol.